### DISCUSSION TITLE
How to use catchup? Catchup doesn't work - there is no slider to move to change time position of the show.

### DISCUSSION THREAD

**Discussion by Alexander-Egistov** [4gray/iptvnator#399] [Q&A]
Catchup does not work - I can load a program from yesterday, but I can not change the time position within the program.

My IPTV provider has catchup and it works on multiple android IPTV programs, yet not on this one.

If I click on the EPG on the right, get all list of all the previous shows and click on one - it starts playing, BUT I can not wind it forward into the middle of the show or manipulate the currently played time beyond the few seconds of the buffer.

That is if I load "wheeler dealers" from yesterday on national geographic I can not skip to the middle of the show, there is simply no way, at least no obvious way to move to the middle of it.

**Comment by BESTS-IPTV** [comment]
the catchup feature it work depend on the service your are using try this <a rel="dofollow" href="https://oneclicktv.net">iptv service</a> they have catchup feature in there subscription

**Reply by Alexander-Egistov** [reply]
the IPTV service provides catchup.
catchup works perfectly in android IPTV applications
but in this application, I can only choose a show from a different time, such as from a few hours ago or a few days ago and watch it, BUT I can not change the time within the show, that is I can't click on the middle of the time slider and watch it from the middle - each time I select a show from yesterday or a few hours ago, I need to watch it from the beggining
